README.md

```diff
@@ -354,24 +354,25 @@ There are three configuration parameters for circuit breakers in Semian:
 
 ### Bulkheading
 
-For many applications, circuit breakers are not enough however. This is best
-illustrated with an extreme. Imagine if the timeout for our data store isn't as
-low as 200ms, but actually 10 seconds. For example, you might have a relational data
+For some applications, circuit breakers are not enough.j This is best illustrated
+with an extreme. Imagine if the timeout for our data store isn't as low as
```

Comment:
"with an extreme" feels to me it needs a noun. It could be "with an extreme example".